node/src/components/consensus/consensus_protocol.rs

```diff
@@ -137,6 +137,9 @@ pub(crate) trait ConsensusProtocol<I, C: Context>: Send {
     /// Triggers a queued action.
     fn handle_action(&mut self, action_id: ActionId) -> ProtocolOutcomes<I, C>;
 
+    /// Current instance of consensus protocol is latest era.
+    fn handle_is_current(&self) -> ProtocolOutcomes<I, C>;
```

Comment:
To be honest, this comment and name are a bit cryptic, but I'm not sure what to suggest in their place...

Reply:
Changed the comment in [`4fc38c4` (#1159)](https://github.com/CasperLabs/casper-node/pull/1159/commits/4fc38c4656eadf46a0b1237e4c64449ad0ab5c3e)